variables.tf

```diff
@@ -120,10 +120,21 @@ variable "codestar_notifications_kms_master_key_id" {
   type        = string
 }
 
+variable "create_ecr_repo" {
+  default     = true
+  description = "Creates a deploy pipeline from ECR trigger."
+  type        = bool
+}
+
+variable "ecr_repository_name" {
```

Comment:
we currently have a mix of single value var for ECR like this and the ecr object var, I would refactor this to the first approach and get rid of the object